lib/renderer/api/remote.js

```diff
@@ -60,7 +60,7 @@ var wrapArgs = function (args, visited) {
 
       ret = {
         type: 'object',
-        name: value.constructor.name,
+        name: value.constructor != null ? value.constructor.name : 'Object',
```

Comment:
@chetverikov I tweaked your change a bit here to return an empty name when `constructor.name` is empty since that would seem to be the expected behavior for anonymous classes like `new (class {})`.